lib/spack/spack/installer.py

```diff
@@ -84,6 +84,15 @@
 #: queue invariants).
 STATUS_REMOVED = 'removed'
 
+#: Don't perform an install
+ACTION_NONE = 0
```

Comment:
(no change required) Personal opinion: it is clearer to bundle actions like this into an "enum":

```
class InstallAction(object):
    NONE = 0
    INSTALL = 1
    OVERWRITE = 2
```

Although this follows a similar pattern as `STATUS_*` so no need to change that.

Reply:
I think this is a Python 3.x feature :(

Reply:
You can write what I wrote without 3.x support (it implements an enum as a design pattern). Like I said though this can be left as is.